tests/imgtype.go

```diff
@@ -79,60 +97,74 @@ func main() {
 
 		ref, err := is.Transport.ParseStoreReference(store, image)
 		if err != nil {
-			logrus.Fatalf("error parsing reference %q: %v", image, err)
-		}
-
-		src, err := ref.NewImageSource(systemContext, []string{expectedManifestType})
```

Comment:
I wasn't referring to the Transport.ParseStoreReference(), but the call to ref.NewImageSource on line 85 in the old code.  I didn't see that elsewhere.